sdk/loadtestservice/load-testing-rest/src/beginUploadAndValidateFile.ts

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AbortController, AbortSignalLike } from "@azure/abort-controller";
+import { CancelOnProgress, OperationState, SimplePollerLike } from "@azure/core-lro";
+import { FileUploadAndValidatePoller, PolledOperationOptions } from "./models";
+import { AzureLoadTestingClient } from "./index.js";
+import {
+  TestGetFile200Response,
+} from "./responses";
+import { isUnexpected } from "./isUnexpected";
+import { ReadStream } from "fs";
+
+/**
+ * Uploads a file and creates a poller to poll for validation.
+ * @param client - The Load Testing client.
+ * @param options - The operation options.
+ * @returns A poller which can be called to poll until completion of the job.
+ */
+export async function beginFileValidation(
+  client: AzureLoadTestingClient,
+  testId: string,
+  fileName: string,
+  file: ReadStream,
```

Comment:
same question here about passing all those parameters instead of just the response.

Reply:
same as testrun, that we want to have a function, where it calls with the parameters similar to other language SDKs for load testing.